### PR TITLE
[FLCRM-9641] Create task location field

### PR DESCRIPTION
**Pull request by andres-martin**
https://fulcrumapp.atlassian.net/browse/FLCRM-9641

### CODE REVIEW

**Review by osbornk — DISMISSED** [review]

**Comment by toddrun** [comment]
@andres-martin , I'm curious why you didn't add a `DirectionsField` to form-v3.json too?

File: dist/fulcrum-schema.min.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function _defineProperties(target, props) { for (var i = 0; i < props.length; i+
 
 function _createClass(Constructor, protoProps, staticProps) { if (protoProps) _defineProperties(Constructor.prototype, protoProps); if (staticProps) _defineProperties(Constructor, staticProps); Object.defineProperty(Constructor, "prototype", { writable: false }); return Constructor; }
 
-var DATA_ELEMENTS = ['TextField', 'ChoiceField', 'ClassificationField', 'YesNoField', 'PhotoField', 'VideoField', 'AudioField', 'SignatureField', 'BarcodeField', 'DateTimeField', 'TimeField', 'Repeatable', 'AddressField', 'HyperlinkField', 'CalculatedField', 'RecordLinkField', 'AttachmentField', 'CheckboxField', 'DynamicField'];
+var DATA_ELEMENTS = ['TextField', 'ChoiceField', 'ClassificationField', 'YesNoField', 'PhotoField', 'VideoField', 'AudioField', 'SignatureField', 'BarcodeField', 'DateTimeField', 'TimeField', 'Repeatable', 'AddressField', 'HyperlinkField', 'CalculatedField', 'RecordLinkField', 'AttachmentField', 'CheckboxField', 'DynamicField', 'DirectionsField'];
```
Comment (chrisworkmode) [inline dist/fulcrum-schema.min.js:15]: This is more of a naming thing, but should we try to go more generic with this field? Like Place Or Location?

Comment (osbornk) [reply dist/fulcrum-schema.min.js:15]: I was thinking that as well. The data is describing a location. In tasks, we are using the location to provide directions. So, the actual type should be LocationField. And then the field name in the Task schema could be called Directions.

Comment (alejo-vasquez) [reply dist/fulcrum-schema.min.js:15]: I agree, I think we could call it _place_ to avoid confusion with the _location_ of the metadata

Comment (andres-martin) [reply dist/fulcrum-schema.min.js:15]: done!

**Review by osbornk — DISMISSED** [review]

**Comment by jahndis** [comment]
It looks like `yarn test` is failing. Might double check if that hilights any issues.

**Comment by andres-martin** [comment]
> It looks like `yarn test` is failing. Might double check if that hilights any issues.

yeah I fixed the failing tests

**Review by osbornk — APPROVED** [review]

**Review by jahndis — APPROVED** [review]

**Pull request merged**